src/internal/data/implementations.go

```diff
@@ -136,3 +137,30 @@ func (col BaseCollection) DoNotMergeItems() bool {
 func (col BaseCollection) Opts() control.Options {
 	return col.opts
 }
+
+// -----------------------------------------------------------------------------
+// tombstoneCollection
+// -----------------------------------------------------------------------------
+
+func NewTombstoneCollection(
+	prev path.Path,
+	opts control.Options,
+) *tombstoneCollection {
+	return &tombstoneCollection{
+		BaseCollection: NewBaseCollection(nil, prev, nil, opts, false),
+	}
+}
+
+// tombstoneCollection is a collection that marks a folder (and folders under it
+// if they aren't explicitly noted in other collecteds) as deleted. It doesn't
+// contain any items.
+type tombstoneCollection struct {
+	BaseCollection
+}
+
+func (col *tombstoneCollection) Items(context.Context, *fault.Bus) <-chan Item {
```

Comment:
would it be easier to return Nil here?

Also, you might want to add a comment explaining that this is always expected to be a no-op, and why.

Reply:
ah yea, nil should work. I sort of forgot that range loops play nice with nil heh

There is a comment on the struct definition that states it never has items, but I'll add something to this function as well